src/commands/update-wcw.ts

```diff
@@ -0,0 +1,43 @@
+import { CommandInteraction, CacheType, User } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CONSTANTS } from '../constants';
+import { addTopic } from '../lib/addTopic';
+import logger from '../utils/logger';
+
+export const data = new SlashCommandBuilder()
+    .setName('update-wcw')
+    .setDescription('Add a topic for Water Cooler Wednesdays 💦')
+    .addStringOption(option =>
+        option.setName('topic').setDescription('Your topic').setRequired(true),
+    );
+
+export async function execute(interaction: CommandInteraction<CacheType>) {
+    const topic = interaction.options.getString('topic') as string;
+    const interactionAuthor = interaction.member?.user as User;
+
+    if (!CONSTANTS.MODERATORS.includes(interactionAuthor.id)) {
```

Comment:
This isn't maintainable long term. Instead, you can check if the author's roles contains the Moderator role. This means we won't have to redeploy this code whenever Mods change.